charts/newrelic-infrastructure-v3/templates/_helpers.tpl

```diff
@@ -188,3 +196,121 @@ Returns Custom Attributes even if formatted as a json string
 {{- define "newrelic.customAttributes" -}}
 {{- merge (include "newrelic.customAttributesWithoutClusterName" . | fromJson) (dict "clusterName" (include "newrelic.cluster" .)) | toJson }}
 {{- end -}}
+
+
+{{/*
+Returns controlPlane component taking into account legacy values
+*/}}
+{{- define "newrelic.compatibility.control-plane" -}}
+enabled: true
+{{- with .autodiscover }}
+autodiscover:
+{{ . | toYaml}}
```

Comment:
Does this not need `indent`?

Reply:
without it it was rending the list as:
```
autodiscover:
- item 1
- item2
```

cleaner with indentation, you are right